ct.yaml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+chart-dirs:
+  - charts
+chart-repos:
+  - openebs-ndm=https://openebs.github.io/node-disk-manager
```

Comment:
Is this the repo from which the charts are fetched for testing?

Reply:
This is the repo from where the charts are installed. For testing in CI -- it is taken from the repo artifacts.

Reply:
this is only required if you have dependencies using https://openebs.github.io/node-disk-manager - else the charts in this repo will be used.